js/react_native/ios/OnnxruntimeModule.mm

```diff
@@ -87,6 +90,14 @@ - (NSDictionary *)loadModel:(NSString *)modelPath options:(NSDictionary *)option
     sessionInfo = new SessionInfo();
 
     Ort::SessionOptions sessionOptions = [self parseSessionOptions:options];
+
+    /*
+      Optional call of to enable usage of ort extensions custom ops
+    */
+    // if (RegisterCustomOps(sessionOptions, OrtGetApiBase()) != nullptr) {
```

Comment:
Open to suggestions for enable optional compile here for extensions call.

seems like this macro won't work at this level 

https://github.com/microsoft/onnxruntime/blob/62a4c02a1d50413d99b6a2fc79aac68886c6c6e4/java/src/main/native/ai_onnxruntime_OrtSession_SessionOptions.c#L123-L126